databases/week3/hw.sql

```diff
@@ -0,0 +1,217 @@
+
+
+CREATE TABLE Meal(
```

Comment:
It's good practice to use plural names in database tables because they're used to store rows (lists) of things